src/table/utils/accessibility-utils.ts

```diff
@@ -211,9 +211,9 @@ export const announceSelectionState = (
   }
 };
 
-export const copyCellValue = async (cell: Cell) => {
+export const copyCellValue = async (value?: string) => {
```

Comment:
why is value optional? When defined like this, the call `copyCellValue()` would be valid, but then here is no value to be copied. If you change to use the text from the event it might always be there?